docker/binding-to-0-0-0-0-in-rails.md

```diff
@@ -0,0 +1,29 @@
+# Binding to 0.0.0.0 in Rails
+
+On OS X, [boot2docker] is an installable VM host to run Docker locally (OS X
+doesn't natively support Linux Containers). In combination with [fig], it's
+pretty straightforward to get a Rails application running on [Docker].
+
+[boot2docker]: http://boot2docker.io/
+[fig]: http://www.fig.sh/
+[docker]: https://www.docker.com/
+
+To access the application, you'll visit `$(boot2docker ip):3000` (or
+whatever port your Rails application is running on) and things should work
+fine. However, I'd recently tried setting up Docker in a Rails app that was
+running WEBrick and wasn't able to access the application at all when running
+`rails server`. I'd recalled using [Thin] previously just fine with `thin
```

Comment:
This sentence is a little awkward. Perhaps something along the lines of:

> I recalled not running into this issue when using Thin...

or 

> I hadn't run into this issue when using Thin...

or 

> I'd previously used Thin and hadn't run into this problem...